src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputNoLineWrapBad.java

```diff
@@ -1,15 +1,17 @@
 package com.puppycrawl.tools.
     checkstyle.checks.whitespace;
 
-import com.puppycrawl.tools.checkstyle.TreeWalker;
+import static java.math.
+	BigInteger.ZERO;
 
-import javax.accessibility.
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
+import javax.accessibility.
+		AccessibleAttributeSequence;
```

Comment:
@romani There is an issue I face everytime  the order of imports in `InputNoLineWrapBad` is changed. The travis ci fails at Subtask No. 16, which says that static should be in its own group
What i do to pass the test
```java
package com.puppycrawl.tools.
    checkstyle.checks.whitespace;

import static java.math.
	BigInteger.ZERO;

import com.puppycrawl.tools.checkstyle.TreeWalker;
import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
import javax.accessibility.
		AccessibleAttributeSequence;

public class
    InputNoLineWrapBad {

	public void
	    fooMethod() {
		final int
		    foo = 0;
	}
}

enum
    FooFoo {
}

interface
    InterFoo {}
```
Where the imports are in placed as mentioned here - https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

But this what you saying i shouldn't do...Please help in this regard